docs/USING.rst

```diff
@@ -50,3 +50,39 @@ to hard-disk booting the installed system.
 This is, as the filepath states, merely an example of how you might deploy to
 physical nodes.  Existing deployment systems such as Canonical's MAAS have also
 been run succesfully.
+
+
+Usage Examples
+=================
+
+Included herewith are some examples about using cli and api calls. 
+
+
+
+1) Register a switch with HaaS:
+-------------------------------
+
+Eg> Switch name: mockswitch01
+     hostname: switchhost01
```

Comment:
Host is lowercase in your hostname here, but uppercase in the arguments for the call. Please be consistent one way or the other. I know its not big, but no reason to potentially confuse people that a hostname needs to be camel case.